### PR TITLE
Added Bulma CSS, page routing, and pages directory

### PR DESCRIPTION
**Pull request by JakeBrown**
This should close issue #28. Let me know if you have any questions.

### CODE REVIEW

File: front-end/src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+            SignInPage
+          </Link>
+          <Link className="navbar-item" to="upload">
+            UploadPage
+          </Link>
+          <div className="navbar-item">
+            <span className="tag">{myName}</span>
+          </div>
+        </div>
+      </nav>
```
Comment (cskeogh) [inline front-end/src/App.tsx:34]: Is the navbar is just for developer/debugging purposes? (I don't think the final production build should have a navbar)

File: front-end/src/components/NameForm.tsx [hunk 2]
```diff
@@ -5,37 +5,43 @@ function NameForm() {
...
-    if (setMyName != null)
-    {
+    if (setMyName != null) {
       setMyName(name);
     }
   }
 
-  function handleChange(event : React.ChangeEvent<HTMLInputElement>) {
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
     console.log(event);
```
Comment (cskeogh) [inline front-end/src/components/NameForm.tsx:16]: This is formatting change on unrelated code. Happy for reformatting, but it is easier to review if you put reformatting under a separate issue like 'reformat code'. I'm happy to let this through, but next time a separate issue?

**Review by JoshuaBuditama — APPROVED** [review]
Everything seems to be working fine.
Ready to be merged.

**Review by cskeogh — APPROVED** [review]
Approve. Note my comments below.

File: front-end/src/components/NameForm.tsx (hunk 2 above)
Comment (JakeBrown) [reply front-end/src/components/NameForm.tsx:16]: Yep good point! Will do.

File: front-end/src/App.tsx (hunk 1 above)
Comment (JakeBrown) [reply front-end/src/App.tsx:34]: Yeah, really just there to provide an example of how the developer should navigate between screens.
